src/main.rs

```diff
@@ -60,6 +62,10 @@ struct Opt {
     #[clap(short = 'w', long, default_value = "postgres")]
     pass: String,
 
+    /// extended query mode in postgresql(only valid in postgresql)
+    #[clap(long, action)]
+    extend: bool,
```

Comment:
I think we can add a different engine `postgresql-extended` instead of adding the option

Reply:
do you mean create a different bin using different engine, and we can use different bin to run test in different mode. such as: 
```
sqllogictest 'e2e_test/*.slt' (simeple query)
postgresql-extended 'e2e_test/*.slt' (extended query)
```

Reply:
```
sqllogictest a.slt --engine postgresql
sqllogictest a.slt --engine mysql
sqllogictest a.slt --engine postgresql-extended
```

Reply:
cool!